docs/azure/sdk/snippets/aspnetcore-guidance/BlazorSample/Program.cs

```diff
@@ -0,0 +1,49 @@
+using BlazorSample.Components;
+using Azure.Identity;
+using Azure.Messaging.ServiceBus;
+using Microsoft.Extensions.Azure;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddRazorComponents();
+
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    // Register a client for each Azure service using inline configuration
+    clientBuilder.AddSecretClient(new Uri("<key_vault_url>"));
+    clientBuilder.AddBlobServiceClient(new Uri("<storage_url>"));
+    clientBuilder.AddServiceBusClientWithNamespace(
+        "<your_namespace>.servicebus.windows.net");
+
+    // Register a subclient for each Azure Service Bus Queue
+    var queueNames = new string[] { "queue1", "queue2" };
+    foreach (string queue in queueNames)
+    {
+        clientBuilder.AddClient<ServiceBusSender, ServiceBusClientOptions>(
+            (_, _, provider) => provider.GetService<ServiceBusClient>()
+                    .CreateSender(queue)).WithName(queue);
```

Comment:
It would be good to show how to get these "named" clients out of DI. Step 4 "Inject the registered services into your ASP.NET Core app components, services, or API endpoint:" just shows how to get the `BlobServiceClient`.

Reply:
We should add this.   It was a source of confusion with the article that we had consistent feedback on.    I'd suggest at least bringing over the content from the snippet [here]9https://learn.microsoft.com/en-us/dotnet/azure/sdk/dependency-injection?tabs=web-app-builder#use-the-registered-clients).

Reply:
It might also be helpful to give an explanation of when a user might want to use `WithName`.

Reply:
@alexwolfmsft I believe this question is answered at https://learn.microsoft.com/en-us/dotnet/azure/sdk/dependency-injection?tabs=web-app-builder#configure-multiple-service-clients-with-different-names.